onchain/src/art_peace.cairo

```diff
@@ -424,4 +435,19 @@ pub mod ArtPeace {
             }
         }
     }
+
+    #[generate_trait]
+    impl InternatImpl of InternalTrait {
+        fn increase_day_index(ref self: ContractState) {
+            let block_timestamp = get_block_timestamp();
+            let start_day_time = self.start_day_time.read();
+
+            if block_timestamp > start_day_time + day_time {
```

Comment:
Using an `assert` instead of a conditional would be preferred here

Reply:
true, definitely! thank you

Reply:
my `if` does the job to prevent updating the day index prematurely, but yeah better to make it revert to avoid useless successful tx